src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.CodeAnalysis.CSharp
     /// The base class for all symbols (namespaces, classes, method, parameters, etc.) that are 
     /// exposed by the compiler.
     /// </summary>
-    [DebuggerDisplay("{GetDebuggerDisplay(), nq}")]
+    //   [DebuggerDisplay("{GetDebuggerDisplay(), nq}")]
     internal abstract partial class Symbol : ISymbol, IFormattable
```

Comment:
:(

Reply:
Sorry let me undo this. This was crashing the debugger and I had to comment out to figure out the bug I'd added. 

---
In reply to: [274624904](https://github.com/dotnet/roslyn/pull/34891#discussion_r274624904) [](ancestors = 274624904)